### PR TITLE
unpatch DRE part upgrades

### PR DESCRIPTION
**Pull request by Schnobs**
A DRE patch ( GameData/DeadlyReentry/DeadlyReentry-Upgrades.cfg )  lowers the cockpit's heat tolerance to 450/450K, rendering it unsafe at any speed. Same patch would also add a part upgrade which doesn't work on the RP-0 tech tree.

This here cleans up the fallout.

On second thought, maybe it would be better to add a NOT[RealismOverhaul] to the original DRE patch?

### CODE REVIEW

**Comment by raidernick** [comment]
This looks like it changes the same thing as the other PR? Can you remove the patch for the tank changes because as @pap1723 said it should be that way on purpose. Also, not sure about the heat tolerances for this part as I never use stock parts at all, @PhineasFreak is the heat tolerance even broken on this, or was it also done to prevent people from using it as an orbital vehicle in early game?

**Comment by PhineasFreak** [comment]
The specific cockpit is definitely not intended for LEO reentries. We have specific (cloned) parts for LEO-rated spaceplanes, like the **RO-Mk1Cockpit**.

**Comment by Schnobs** [comment]
(Sigh) let me outline the problem once more:
DRE applies patches to the Mark1Cockpit which override the RO config. That's override, not overwrite: the part stats still show the RO values. But in flight, the invisible DRE values are in effect.

File: GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Spaceplanes.cfg [hunk 1]
```diff
@@ -1696,7 +1696,7 @@
 		name = ModuleFuelTanks
 		volume = 100
 		basemass = -1
-		type = ServiceModule
+		type = Fuselage
```
Comment (raidernick) [inline GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Spaceplanes.cfg]: This is what I'm talking about. Why is this here, it changes the same thing the other pr does which is resource related for tacls. That pr is not going to be accepted anyway based on what pap said.

**Comment by PhineasFreak** [comment]
If all parts are affected then if will be much easier and cleaner to just change the base DRE configs so that they will not apply when RO is installed.
